Lib/test/test_htmlparser.py

```diff
@@ -787,5 +789,17 @@ def test_weird_chars_in_unquoted_attribute_values(self):
                             ('starttag', 'form',
                                 [('action', 'bogus|&#()value')])])
 
+
+class TestInheritance(unittest.TestCase):
+
+    @patch("_markupbase.ParserBase.__init__")
+    @patch("_markupbase.ParserBase.reset")
+    def test_init(self, super_init_method, super_reset_method):
```

Comment:
The test name should be more meaningful, and the two args are swapped:
```suggestion
    @patch("_markupbase.ParserBase.__init__")
    @patch("_markupbase.ParserBase.reset")
    def test_base_class_methods_called(self, super_reset_method, super_init_method):
```